server/keyshare/keyshareserver/conf.go

```diff
@@ -63,6 +63,14 @@ type Configuration struct {
 	registrationEmailTemplates map[string]*template.Template
 
 	VerificationURL map[string]string `json:"verification_url" mapstructure:"verification_url"`
+	// number of tries allowed on pin before we start with exponential backoff
+	MaxPinTries int `json:"max_pin_tries" mapstructure:"max_pin_tries"`
```

Comment:
This is a chore security parameter, and I doubt whether we want to make this configurable. There is not really another logical value than the one we had hardcoded in my view. This means that making this setting configurable only introduces a risk on misconfiguration. I'd keep this one constant.